tests/test_image_processing_common.py

```diff
@@ -129,176 +129,214 @@ def prepare_video_inputs(
 
 class ImageProcessingTestMixin:
     test_cast_dtype = None
+    image_processing_class = None
+    fast_image_processing_class = None
+    image_processors_list = None
+    test_slow_image_processor = True
+    test_fast_image_processor = True
```

Comment:
should it not default to false?

Reply:
I want it to default to `True` such that as soon as someone add the fast image processor to the tester (which I'm not sure how to encourage) the tests will automatically run